apps/web/screens/OpenDataSubPage/OpenDataSubPage.tsx

```diff
@@ -0,0 +1,198 @@
+                  preTextIconType="filled"
+                  variant="text"
+                  size="small"
+                  as="span"
+                >
+                  Til baka
+                </Button>
+              </Link>
+            </Box>
+            <Box
+              padding={[2, 2, 4]}
+              background="purple100"
+              borderRadius="large"
+              marginBottom={3}
+            >
+              <Text variant="eyebrow" color="purple600">
+                Þjónustuaðili
+              </Text>
+              <Box display="flex" alignItems="center">
+                <Box display="inlineFlex" flexGrow={1}>
+                  <Text variant="h3" color="purple600">
+                    Rannís
+                  </Text>
+                </Box>
+              </Box>
+            </Box>
+            <Box
+              padding={[2, 2, 4]}
+              background="blueberry100"
+              borderRadius="large"
+            >
+              <Text variant="eyebrow" color="blueberry600">
+                Tengt efni
+              </Text>
+              <Box display="flex" alignItems="center">
+                <Box display="inlineFlex" flexGrow={1}>
+                  <Text color="blueberry600">Tæknisjóður</Text>
+                </Box>
+              </Box>
+            </Box>
+          </GridColumn>
+          <GridColumn span="8/12" offset="1/12">
+            <Box marginBottom={3}>
+              <Breadcrumbs
+                items={[
+                  {
+                    title: 'Ísland.is',
+                    href: '/',
+                  },
+                  {
+                    title: 'Gagnatorg',
+                    href: '/gagnatorg',
+                  },
+                  {
+                    title: 'Rannís',
+                  },
+                ]}
+                renderLink={(link) => {
+                  return (
+                    <NextLink {...linkResolver('opendatapage')} passHref>
+                      {link}
+                    </NextLink>
+                  )
+                }}
+              />
+            </Box>
+            <Box marginBottom={8}>
+              <Text variant="h1">Mælaborð</Text>
+            </Box>
+            <Box
+              background="roseTinted100"
+              padding={[2, 2, 4]}
+              borderRadius="large"
+              marginBottom={3}
+            >
+              <Inline space={2}>
+                <Icon
+                  color="purple400"
+                  icon="cellular"
+                  size="small"
+                  type="outline"
+                />
+                <Text variant="eyebrow">Sjóðir Rannís</Text>
+              </Inline>
+            </Box>
+            <Box marginBottom={3}>
+              <GridRow>
+                <GridColumn span="6/12">
+                  <StatisticsCard
+                    title="Samningar/umsóknir 2020-21"
+                    description="127/745"
```

Comment:
No, the plan was to get reviews on the open data page first and the cms connections since this is my first time working in this project and then apply my new found knowledge to the sub page. Its all hard coded now but will not be tomorrow :)

Reply:
> Left some comments, a lot of it is just nit-picking.
> 
> Looks good overall, but there are a lot of hard-coded labels and values, and some code blocks commented out. Are you still working on this? or were you requested to deploy a demo or something like that?

And yes was asked to make a demo by tomorrow and I am still working on this, goal is to remove most of the hard coded values today